linode/manager
# upcoming: [M3-7816] - Hide "Switch Account" buttons if child_account_access is false

## Description 📝
For parent account users (non-admin), we need to ensure Switch Account button doesn't show if they don't have the child_account_access grant.

## Changes  🔄
- Checking for `child_account_access` as part of condition
- Added unit test for UserMenu

## Target release date 🗓️
03/04/2024

## Preview 📷
| Before  | After   |
| ------- | ------- |
| ![Screenshot 2024-02-28 at 12 06 24 PM](https://github.com/linode/manager/assets/125309814/b01c03ce-aa02-4b91-baad-badb431b6657) | ![Screenshot 2024-02-28 at 12 06 47 PM](https://github.com/linode/manager/assets/125309814/fcfbe0fd-c8e2-4d67-a196-21472f0a0952) |
| ![Screenshot 2024-02-28 at 12 06 31 PM](https://github.com/linode/manager/assets/125309814/8b8bfcf6-6c7a-4d5d-bcd0-b4d440452b76) | ![Screenshot 2024-02-28 at 12 06 52 PM](https://github.com/linode/manager/assets/125309814/6fb89dfc-53b9-4a9d-8a79-00e9b3ce6841) |

## How to test 🧪

### Prerequisites
- Create a parent account
- Create a non-admin user on the account
- Ensure this new users `child_account_access` grant is set to `false`

### Reproduction steps
- Go to http://localhost:3000/account/billing
- Observe that without changes you can see the Switch Account buttons

### Verification steps
- Checkout local changes or preview link
- Observe that the Switch Account button is not rendered

## As an Author I have considered 🤔

*Check all that apply*
